pkg/rbd/rbd_util.go

```diff
@@ -53,21 +63,30 @@ type rbdVolume struct {
 	Mounter            string `json:"mounter"`
 	DisableInUseChecks bool   `json:"disableInUseChecks"`
 	ClusterID          string `json:"clusterId"`
+	// RequestName is the CSI generated volume name for the rbdVolume
+	RequestName string `json:"requestName"`
 }
 
+// rbdSnapshot represents a CSI snapshot and its RBD snapshot specifics
 type rbdSnapshot struct {
-	SourceVolumeID     string `json:"sourceVolumeID"`
-	VolName            string `json:"volName"`
-	SnapName           string `json:"snapName"`
-	SnapID             string `json:"sanpID"`
-	Monitors           string `json:"monitors"`
-	MonValueFromSecret string `json:"monValueFromSecret"`
-	Pool               string `json:"pool"`
-	CreatedAt          int64  `json:"createdAt"`
-	SizeBytes          int64  `json:"sizeBytes"`
-	AdminID            string `json:"adminId"`
-	UserID             string `json:"userId"`
-	ClusterID          string `json:"clusterId"`
+	// SourceVolumeID is the volume ID of VolName, that is exchanged with CSI drivers
+	SourceVolumeID string `json:"sourceVolumeID"`
+	// VolName is the name of the RBD image, that is this rbdSnapshot's source image
```

Comment:
Nit: confusing -- ```VolName``` elsewhere is the CSI volume name not the RBD image name. Rename to ```ImageName``` if that's it's intended use?